app/experimenter/experiments/models/nimbus.py

```diff
@@ -195,6 +195,24 @@ def __str__(self):
         return self.name
 
 
+class NimbusDocumentationLink(models.Model):
+    experiment = models.ForeignKey(
+        NimbusExperiment,
+        related_name="documentation_links",
+        on_delete=models.CASCADE,
+    )
+    title = models.CharField(max_length=255, null=False)
+    link = models.URLField(max_length=255, null=False)
+
+    class Meta:
+        verbose_name = "Nimbus Documentation Link"
+        verbose_name_plural = "Nimbus Documentation Links"
+        ordering = ("id",)
```

Comment:
Given that we delete/recreate every link on every save, id won't be a stable ordering, we should probably use `title`?

Reply:
Well, they should be re-created with the order in which they came from the form, which I think should result in sequential IDs? Not the originals but still in order

Reply:
In my experience you can't depend on database id ordering being stable, so I typically use something in the data itself.

Reply:
We should keep our eye out if the ordering in the UI becomes wonky, or if tests intermittently fail.